insanely_fast_whisper_rocm/cli/commands.py

```
@@ -239,39 +497,66 @@ def _handle_output_and_benchmarks(
     export_format_explicit: bool,
     benchmark_enabled: bool,
     benchmark_extra: tuple[str, ...],
+    benchmark_flags: dict[str, Any] | None,
+    benchmark_gpu_stats: dict[str, Any] | None,
     temp_files: list[Path],
+    progress_cb: ProgressCallback | None = None,
+    quiet: bool = False,
```

Ruff (0.14.7)

Unused function argument: `quiet`

(ARG001)